source/_integrations/compensation.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: Compensation
+description: Instructions on how to integrate compensation sensors into Home Assistant.
+ha_category:
+  - Utility
+  - Sensor
+ha_iot_class: Local Push
```

Comment:
This integration calculates its states based on existing data.

```suggestion
ha_iot_class: Calculated
```

Reply:
Yep, I assumed that was 'local push'.  Makes more sense as calculated.